test/sandbox/app/views/integration_examples/empty_slot.html.erb

```diff
@@ -2,3 +2,8 @@
   <% component.slot(:title) do %>
   <% end %>
 <% end %>
+
+<%= render(EmptySlotV2ComponentWithSlim.new) do |component| %>
```

Comment:
At first I added an entirely new route, view, and component (`/nil_slot`, `nil_slot.html.erb`, and `NilSlotComponent.rb`), but I think this is clearer: I copied the EmptySlotComponent, updated it to use SlotV2, and then re-used the same integration test to test both view components. WDYT?